src/main.cpp

```diff
@@ -6005,6 +6005,9 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
         {
             boost::this_thread::interruption_point();
 
+            if (!(addr.nServices & NODE_NETWORK))
+                continue;
+
```

Comment:
I assume with this code (which is still in Core's mainline) then that pruned nodes don't get incoming connections?  Does seem a bit harsh and unclear why this was put in.  I'll drop this commit.
@ptschip they'll get new addresses from outgoing connections presumably.  When you say we don't get as many incoming connections do you mean to pruned nodes?